data-validation/use-cases.md

```diff
@@ -2,15 +2,148 @@
 
 ## Introduction
 
-Server-side data validation is an essential factor to the seamless interoperability promised by Solid. Two applications that manipulate the same kind of data, developed with no knowledge of each other, should be able to write that data without breaking each other. Using common linked data vocabularies is not enough to satisfy this fundamental need. Data Shape languages like [ShEx](http://shex.io/shex-semantics) and [SHACL](https://www.w3.org/TR/shacl/) help us solve this.
+Server-side data validation is an essential factor to the seamless interoperability promised by Solid. Multiple applications that manipulate the same kind of data, developed with no knowledge of each other, should be able to write that data without breaking each other. This requires all applications to conform to a common schema. Data Shape languages like [ShEx](http://shex.io/shex-semantics) and [SHACL](https://www.w3.org/TR/shacl/) help us solve this.
```

Comment:
I actually do not agree with the phrasing above.

If all applications conform to a common schema, then we need centralized agreement, and can solve it with much easier tooling than RDF.

The key challenge is how to interoperate in absence of slow central agreement processes.